src/ocrd/processor/base.py

```diff
@@ -287,17 +314,19 @@ def process_workspace(self, workspace: Workspace) -> None:
             self.verify()
             try:
                 # FIXME: add page parallelization by running multiprocessing.Pool (#322)
-                for input_file_tuple in self.zip_input_files(on_error='abort'):
+                for input_file_tuple in self.zip_input_files(on_error='abort', require_first=False):
                     # FIXME: add error handling by catching exceptions in various ways (#579)
                     # for example:
                     # - ResourceNotFoundError → use ResourceManager to download (once), then retry
                     # - transient (I/O or OOM) error → maybe sleep, retry
                     # - persistent (data) error → skip / dummy / raise
                     input_files : List[Optional[Union[OcrdFile, ClientSideOcrdFile]]] = [None] * len(input_file_tuple)
```

Comment:
and in the same vain, it might be useful to have `ocrd.typing` containing stuff like

```python
OcrdFileType = Union[OcrdFile, ClientSideOcrdFile]
OcrdPageType = Union[OcrdPage, PcGtsType]
VersionType =  Union[Tuple[int, int, int], Tuple[int, int, int, str]]
```

Reply:
Yes, that would be useful. But why the extra module level `typing`?

Reply:
It doesn't have to be a separate file/module, just somewhere importable. But it has the same potential pitfalls with circular dependencies as in the other cases discussed, so let's focus on the functionality first, cleaner codebase and optimizations later.